config/rapidez/system.php

```diff
@@ -10,6 +10,9 @@
     // Elasticsearch prefix.
     'es_prefix' => env('ELASTICSEARCH_PREFIX', 'rapidez'),
 
+    // Get Magento url from Database
+    'magento_url_from_db' => env('GET_MAGENTO_URL_FROM_DATABASE', false),
```

Comment:
Pros/cons of having this true/false by default?

Reply:
Pro for false:
- no breaking changes by default, only when you opt into this functionality will it behave differently 
- after upgrading you have the time to first get your database in order before enabling this feature.
Pro for true:
- You get a much better experience with url handling out of the box since you no longer need to define the urls in Rapidez itself with subfolders working

If your database has the correct urls by default, and you don't have them set using env variables or config.php.
But this is an assumption we'd have to make when defaulting it to true

Reply:
Thanks! Just one thing left; documenting this option here: https://docs.rapidez.io/3.x/configuration.html#base-url